### PR TITLE
Enhance version for geosafe landing page

### PR DESCRIPTION
**Pull request by boney-bun**
Update the current geosafe landing page by adding animated images and changing buttons.
According to #62

Please review the code @dimasciput

### CODE REVIEW

**Comment by dimasciput** [comment]
Could you provide a screenshot or gif animation for this?

**Comment by boney-bun** [comment]
Here is the latest snapshot of the landing page @dimasciput :
enhanceui62
Note: the description text for Geosafe in the first line has changed according to #126

**Comment by dimasciput** [comment]
@boney-bun 

- Add padding-top and padding-bottom to the image. It's too close with the title and button.
- Make images one size
- If images is mostly white, add border
- Change color of the button to white if the background is blue, and blue if background white

**Comment by boney-bun** [comment]
I have revised the images and layout based on your feedback @dimasciput 
enhanceui62

**Comment by dimasciput** [comment]
@boney-bun next time, could you upload non shrink gif? It's hard to see. Use licecap to create a gif. http://www.cockos.com/licecap/

- Add more padding between image and the texts
- Change to simple arrow button, http://fontawesome.io/icon/arrow-down/
- Match the blue color of the button with the background, use this color ```#2c689c```
- Change hover color to darker original color, not orange

**Comment by boney-bun** [comment]
@dimasciput thanks for your feedback. hope this latest gif would give you a clear picture.
enhancelandingpage62
Note: once we have agreed on the UI, I would need to update the PR.

File: templates/index.html [hunk 1]
```diff
@@ -27,29 +47,49 @@
...
+                                  <div><img src="{% static 'geosafe/img/meta-search.png' %}"/></div>
+                              </div>
+                          </div>
+                      </td>
+                      <td>
+                          <ul style="margin: 5px 50px;">
+                              <li>
+                                  <b>Perform Analysis</b>
+                                  <p>
+                                      Analyze a map according to the given layers (exposure and hazard) and generate
```
Comment (timlinux) [inline templates/index.html]: Perform an InaSAFE analysis using hazard and exposure data hosted on GeoNode.
Visualize the InaSAFE analysis results as a map and infographic and as downloadable PDF reports.

File: templates/index.html [hunk 2]
```diff
@@ -27,29 +47,49 @@
...
+                                      InaSAFE report
+                                  </p>
+                              </li>
+                              <li>
+                                  <b>Analysis List</b>
+                                  <p>
+                                      Contain saved map analysis for future reference
```
Comment (timlinux) [inline templates/index.html]: Browse previously run InaSAFE analysis results and visualize the map and report products for each analysis.

File: templates/index.html [hunk 3]
```diff
@@ -27,29 +47,49 @@
...
+                                  </p>
+                              </li>
+                              <li>
+                                  <b>Metasearch</b>
+                                  <p>
+                                      Search based on Catalog Service for the Web (CSW)
```
Comment (timlinux) [inline templates/index.html]: Search for datasets in other online geospatial data stores that publish their catalogues using the CSW standard.

File: templates/index.html [hunk 4]
```diff
@@ -60,22 +100,42 @@
...
+                                  <div><img src="{% static 'geosafe/img/select-exposure.png' %}"/></div>
+                                  <div><img src="{% static 'geosafe/img/select-impact-layer.png' %}"/></div>
+                                  <div><img src="{% static 'geosafe/img/analysis-being-run.png' %}"/></div>
+                              </div>
+                          </div>
+                      </td>
+                      <td>
+                          <ul style="margin: 5px 50px;">
+                              <li>
+                                  Select an exposure layer (road, structure, population)
```
Comment (timlinux) [inline templates/index.html]: Select an exposure layer (e.g. road, structure, population)

File: templates/index.html [hunk 5]
```diff
@@ -60,22 +100,42 @@
...
+                              </li>
+                              <li>
+                                  Select a hazard layer (flood, tsunami, earthquake, volcano, and volcanic ash)
```
Comment (timlinux) [inline templates/index.html]: Select a hazard layer (e.g. flood, tsunami, earthquake, volcano).

**Review by timlinux — COMMENTED** [review]
Remove the 

* Being run image
* remove the metasearch image
* focus on good looking maps or graphics

**Comment by gubuntu** [comment]
in addition to Tim's comments, ensure there is not visible border or margin around each screenshot or gif

**Pull request merged**